lib/zendesk_apps_tools/translate.rb

```diff
@@ -95,6 +95,14 @@ def self.source_root
     end
 
     no_commands do
+      def fetch_locale_async(locale, app_package)
+        Thread.new do
+          say("Fetching #{locale['locale']}")
```

Comment:
If the output gets minced by the multithreading, you might have to set `$stdout = true` at the start of the command.